utils_test.py

```diff
@@ -16,6 +16,7 @@
 
 # pylint: disable=relative-import
```

Comment:
I noticed that we have added pylint exceptions in other files as well for relative imports as well. So, what should I follow?